### PR TITLE
fix multiple Chinese (Simplified) translation issues

### PR DESCRIPTION
**Pull request by at-wr**

### CODE REVIEW

File: HedgeModManager/Languages/zh-CN.xaml [hunk 1]
```diff
@@ -229,12 +229,12 @@
     <system:String x:Key="GameSonicLostWorld">索尼克 失落世界</system:String>
     <system:String x:Key="GameSonicForces">索尼克 力量</system:String>
     <system:String x:Key="GamePuyoPuyoTetris2">魔法气泡 特趣思 俄罗斯方块 2</system:String>
-    <system:String x:Key="GameTokyo2020">2020东京奥运</system:String>
+    <system:String x:Key="GameTokyo2020">索尼克在 2022 东京奥运会</system:String>
```
Comment (RagdollClash) [inline HedgeModManager/Languages/zh-CN.xaml]: Revert this change, it's for the game on Steam.
https://steamdb.info/app/981890/info/

**Review by RagdollClash — CHANGES_REQUESTED** [review]

File: HedgeModManager/Languages/zh-CN.xaml (hunk 1 above)
Comment (at-wr) [reply HedgeModManager/Languages/zh-CN.xaml]: Sorry, my bad

**Review by JoeyPenis — APPROVED** [review]

**Review by RagdollClash — APPROVED** [review]
LGTM
